app/models/manageiq/providers/embedded_ansible/automation_manager/playbook_runner.rb

```diff
@@ -145,11 +147,32 @@ def queue_signal(*args, deliver_on: nil)
     )
   end
 
-  def playbook
-    ManageIQ::Providers::EmbeddedAnsible::AutomationManager::Playbook.find(options[:playbook_id])
+  def temp_configuration_script
+    ManageIQ::Providers::EmbeddedAnsible::AutomationManager::ConfigurationScript.new(
+      :manager     => playbook.manager,
+      :manager_ref => options[:job_template_ref],
+      :variables   => {}
+    )
   end
 
-  def tower_job
-    ManageIQ::Providers::EmbeddedAnsible::AutomationManager::Job.find(options[:tower_job_id])
+  def delete_inventory
+    return unless options[:inventory]
+    playbook.manager.with_provider_connection do |connection|
+      connection.api.inventories.find(options[:inventory]).destroy!
+    end
+    true
```

Comment:
This shouldn't be needed as the `destroy!` call will return the destroyed instance.